life_expectancy/load_save_data.py

```diff
@@ -1,23 +1,77 @@
 """This model loads and saves the data"""
-from pathlib import Path
+#pylint: disable=R0903
```

Comment:
**Suggestion** Maybe it makes sense to set this configuration globally at the pyproject.toml file

Reply:
Done! Thanks :)